### PR TITLE
Add json format

### PR DESCRIPTION
**Pull request by Syam**
My contrib to add json format:
```
<quill-editor theme="snow" format="json"></quill-editor>
```

Regards,

### CODE REVIEW

File: src/quill-editor.component.ts [hunk 1]
```diff
@@ -203,6 +209,12 @@ export class QuillEditorComponent
         this.quillEditor.setContents(this.content, 'silent');
       } else if (this.format === 'text') {
         this.quillEditor.setText(this.content, 'silent');
+      } else if (this.format === 'json') {
```
Comment (KillerCodeMonkey) [inline src/quill-editor.component.ts]: ❤️

File: src/quill-editor.component.ts [hunk 2]
```diff
@@ -122,18 +123,23 @@ export class QuillEditorComponent
...
-  }
+  };
+
   @Input()
-  valueSetter = (quillEditor: any, value: any, format: 'object' | 'html'): any => {
+  valueSetter = (quillEditor: any, value: any, format: 'object' | 'html' | 'json'): any => {
     if (this.format === 'html') {
       return quillEditor.clipboard.convert(value);
+    } else if (this.format === 'json') {
+      return JSON.parse(value);
```
Comment (KillerCodeMonkey) [inline src/quill-editor.component.ts]: please add try catch ;)

Comment (Syam) [reply src/quill-editor.component.ts]: Ok, added more exceptions

**Comment by Syam** [comment]
Hello, is it ok for you with the last change ?

**Comment by KillerCodeMonkey** [comment]
Yeah, but no time to merge it and build a New Version.

Maybe at the end of the week

Jean Christophe Huwette <notifications@github.com> schrieb am Di., 24. Juli
2018, 02:35:

**Pull request merged**